articles/quickstart/spa/angular/02-calling-an-api.md

```diff
@@ -158,6 +158,3 @@ Your application will show an empty JSON object if you have not set any `user_me
 
 Please refer to the the [Auth0 API quickstarts](https://auth0.com/docs/quickstart/backend) to learn how to integrate Auth0 with your backend platform.
 
-:::note
```

Comment:
Same situation here where the note is removed here but not added again.

Reply:
I also added this one on the new commit.